src/status_im2/contexts/chat/messages/message/delete_message/events.cljs

```diff
@@ -0,0 +1,112 @@
+(ns status-im2.contexts.chat.messages.message.delete-message.events
+  (:require [status-im.chat.models.message-list :as message-list]
+            [status-im.chat.models.pin-message :as models.pin-message]
+            [status-im.ethereum.json-rpc :as json-rpc]
+            [status-im.utils.datetime :as datetime]
```

Comment:
should we look to migrate all of these imports to status-im2 as well? Or do it in follow up work?